pages/services/cassandra/2.6.0-3.11.4/release-notes/index.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes  
+excerpt: Discover the new features, updates, and known limitations in this release of the Cassandra Service
+title: Release Notes 
+menuWeight: 10
+model: /services/cassandra/data.yml
+render: mustache
+---
+# Release Notes for Cassandra Service version 2.6.0-3.11.4
+
+## Upgrades
+
+- Upgraded {{ model.techShortName }} to version `3.11.4`
+- Upgraded SDK library to version `0.56.1`
+
+## New Features
+
+- Added support for both [custom](/services/{{ model.serviceName }}/2.6.0-3.11.4/configuration/#custom-authentication-and-authorization) and {{model.techName}}'s [native](/services/{{ model.serviceName }}/2.6.0-3.11.4/configuration/#native-authentication-and-authorization) authentication and authorization mechanisms 
+- Custom authentication, and authorization are also now supported.
+- Added support for DSS volume profiles 
+- User defined functions are now configurable
```

Comment:
Sync with @ankitcid - we added `custom_domain` support too and he can provide input for docs. Please include code examples!